readme/BREAKING_CHANGES.markdown

```diff
@@ -535,4 +535,26 @@ Import `com.liferay.petra.string.StringBundler` instead of
 This change was made in order to deprecate class
 `com.liferay.portal.kernel.util.StringBundler`
 
+---------------------------------------
+
+### frontend-css-web CSS classes have been removed or replaced
+- **Date:** 2021-Jun-17
+- **JIRA Ticket:** [LPS-127085](https://issues.liferay.com/browse/LPS-127085)
+
+#### What changed?
+
+Many CSS classes have been removed or replaced, you can find an updated list here: https://docs.google.com/spreadsheets/d/1gWdQvLJe8KApcY70lxQb5fuvg6XKdwkfMEBLVPkigfU
```

Comment:
i think we should refer to the module, it's a quick way of reducing the scope to check and remove uncertainty (I am using Clay CSS, should I check the list of classes 1 by 1?)

Reply:
What do you mean?

Reply:
You are only referring to the `frontend-css-web` module in the title. And if you read the What changed is not very clear. `Many CSS classes in frontend-css-web have been removed or replaced` ...